docs/management_api.md

```diff
@@ -43,7 +43,7 @@ you can choose between synchronous or asynchronous call to make sure initial wor
 The asynchronous call returns with HTTP code 202 before trying to create workers.
 
 ```bash
-curl -v -X POST "http://localhost:8081/models?initial_workers=1&synchronous=false&url=https://<s3_path>/squeezenet_v1.1.mar"
+curl -v -X POST "http://localhost:8081/models?initial_workers=1&synchronous=false&url=https://<cloud_OR_noncloud_storage>/squeezenet_v1.1.mar"
```

Comment:
Replace cloud_OR_noncloud_storage with links for squeezenet here https://github.com/pytorch/serve/blob/master/docs/model_zoo.md